man/coredump.conf.xml

```diff
@@ -109,6 +109,20 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>AccessContainer=</varname></term>
+
+        <listitem><para>Controls whether <command>systemd-coredump</command> will attempt to use the mount tree of
+        a process that crashed within a container. Access to the container's filesystem might be necessary to generate
+        a fully symbolized backtrace. If set to <literal>yes</literal>, then <command>systemd-coredump</command> will
+        obtain the mount tree from corresponding mount namespace and will try to generate the stack trace using the
+        binary and libraries from the mount namespace. Defaults to <literal>no</literal>. Note that the coredump might
```

Comment:
I'm not sure what the last sentence is supposed to mean...

Reply:
The point is that even if you have AccessContainer=no, then coredump from container maybe stored to /var/lib/systemd/coredump/ and it reads "might" because whether corefile actually stay in there is also governed by coredump size options, like ExternalSizeMax=.

Reply:
Maybe you want to specify that you are talking about storage on the host's filesystem

Reply:
This still needs to be fixed